pkg/services/blockfetcher/blockfetcher.go

```diff
@@ -192,7 +192,9 @@ func New(chain Ledger, cfg config.NeoFSBlockFetcher, logger *zap.Logger, put fun
 
 func getHeaderSizeMap(chain config.Blockchain) map[int]int {
 	headerSizeMap := make(map[int]int)
-	headerSizeMap[0] = block.GetExpectedHeaderSize(chain.StateRootInHeader, chain.GetNumOfCNs(0))
+	// Genesis header size.
+	headerSizeMap[0] = block.GetExpectedHeaderSize(chain.StateRootInHeader, 0)
+	headerSizeMap[1] = block.GetExpectedHeaderSize(chain.StateRootInHeader, chain.GetNumOfCNs(0))
```

Comment:
And this particular change is not related to this commit, it's a separate bug.

>*: fix GetExpectedHeaderSize for genesis block case 

And it's not `*:`, it's `block:`.